src/org/openjump/core/ccordsys/utils/ProjUtils.java

```diff
@@ -486,6 +487,7 @@ public static SRSInfo getSRSInfoFromLayerSource(RasterImageLayer layer)
         // fallthrough
         if (srsInfo == null) {
             srsInfo = ProjUtils.getSRSInfoFromAuxiliaryFile(fileSourcePath);
+             srsInfo.setSource(AUXFILE_SRS);

```

Comment:
this should be set in getSRSInfoFromAuxiliaryFile() **and only if** the ".aux.xml" file is indeed available and the srs is read from it.

Reply:
I will remove this line as I didn't realize that ProjUtils.getSRSInfoFromAuxiliaryFile() has already a setSource that points to the path of aux file (strangely it fails sometimes, I will inverstigate later)